integration_tests/src/main/python/hash_aggregate_test.py

```diff
@@ -1,3 +1,4 @@
+/enable
```

Comment:
I think this is a type that is causing the build to fail

Reply:
Apologies, I realized after the build failed that I ran with -DskipTests on a non-GPU machine.  I reran with `mvn clean verify -Dcuda.version=cuda10-2` and the tests passed.  I will try rerunning the premerge tests.